tool/jt.rb

```diff
@@ -2002,6 +2003,75 @@ def profile(*args)
     app_open svg_filename
   end
 
+  def graph(*args)
+    test_file = nil
+    method = 'Object#foo'
+    watch = false
+
+    until args.empty?
+      arg = args.shift
+      case arg
+      when '--method'
+        raise if args.empty?
+        method = args.shift
+      when '--watch'
+        watch = true
+      else
+        raise if arg.start_with?('-')
+        raise if test_file
+        test_file = arg
+      end
+    end
+
+    raise unless test_file
+
+    if Gem::Specification.find_all_by_name('seafoam').empty?
+      sh env, 'gem', 'install', 'seafoam'
```

Comment:
`env` doesn't seem defined here.
`gem install` might also be rather slow, how about checking if there is a sibling `seafoam` repo or clone it if not there?

Reply:
I was copying Rubocop which is installed as a gem. I can clone if you think it's important. I'll fix `env` either way.

Reply:
Ah, seafoam is run on the Ruby running JT, typically CRuby, so then it doesn't matter as it'll need to be installed only once in that Ruby

Reply:
It'd be good to set `env = { 'PATH' => "#{File.dirname(ruby)}:#{ENV['PATH']}" }` like for RuboCop, so we make sure the same Ruby running jt is the one running seafoam, and not potentially some other ruby in PATH which `Gem::Specification.find_all_by_name('seafoam')` would not check.

Reply:
I'll do it